social_panel/Gemfile.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem "faraday", "~> 1.0"
+
+gem "rspec", "~> 3.9"
+
+gem "json", "~> 2.3"
+
+gem "webmock", "~>3.8"
```

Comment:
I'd suggest changing the config of your editor ;) It should be adding missing lines at the end of the files for you ;)